numpy/testing/tests/test_utils.py

```diff
@@ -432,6 +437,109 @@ def test_nan_items(self):
                 lambda: self._assert_func(ainf, anan))
 
 
+class TestArrayAssertLess(unittest.TestCase):
+
+    def setUp(self):
+        self._assert_func = assert_array_less
```

Comment:
I don't have any particular feeling concerning this. I was just trying to follow the surrounding style as `TestApproxEqual` is also using it without inheriting from anywhere and not being inherited from. As it doesn't make the code less understandable I would tend to leave it in, but if you feel that it should be dropped, I can also do that.